alacritty/src/migrate.rs

```diff
@@ -0,0 +1,130 @@
+//! Configuration file migration.
+
+use std::fs;
+use std::path::Path;
+
+use toml::Value;
+
+use crate::cli::MigrateOptions;
+use crate::config;
+
+/// Handle migration.
+pub fn migrate(options: MigrateOptions) {
```

Comment:
Seems like the easiest solution to me?

Reply:
I mean, it's a bit weird that we have normal error handling in other sub command and have lazy `eprintln` + `exit` here.

Reply:
I want to keep it this way because it gives us more control over the error messages.

We're not just trying to print errors and bail here, but instead we're trying to have nice human-readable CLI output.

That's why I did it this way, letting Rust's `?` handle error printing is usually worse.